spring-cloud-context/src/main/java/org/springframework/cloud/autoconfigure/RefreshAutoConfiguration.java

```diff
@@ -158,27 +153,6 @@ public String toString() {
 
 	}
 
-	@Configuration(proxyBeanMethods = false)
-	@ConditionalOnClass(name = "javax.persistence.EntityManagerFactory")
-	protected static class JpaInvokerConfiguration implements LoadTimeWeaverAware, InitializingBean {
-
-		@Autowired
-		private ListableBeanFactory beanFactory;
-
-		@Override
-		public void afterPropertiesSet() {
-			String cls = "org.springframework.boot.autoconfigure.jdbc.DataSourceInitializerInvoker";
```

Comment:
`DataSourceInitializerInvoker` was removed in the Spring Boot commit you referenced, correct?

Therefore the `if` statement below never actually gets executed, right?

Reply:
Not exactly, Spring Boot rework DataSource initialization to fix it entirely, it can be proved since `@ConditionalOnClass(name = "javax.persistence.EntityManagerFactory")` never matches since it renamed to `jarkata.persistence.EntityManagerFactory`.